roles/ethereum_node/defaults/main.yaml

```diff
@@ -21,7 +21,6 @@ ethereum_node_cl_ports_http_beacon: 5052
 ethereum_node_cl_ports_grpc_beacon: 4000
 ethereum_node_cl_ports_metrics: 5054
 ethereum_node_cl_checkpoint_sync_enabled: false
-ethereum_node_cl_mev_boost_enabled: false
 
```

Comment:
ethereum_node_mev_boost_enabled: false is missing

Reply:
https://github.com/ethpandaops/ansible-collection-general/blob/323e94697645fd5f11f875bcd9ce2663d0b68d2a/roles/ethereum_node/defaults/main.yaml#L47

Reply:
it was always there, I just missed it before lol